components/organisms/o-microcart.vue

```diff
@@ -29,10 +29,7 @@
               </div>
             </template>
             <template #actions>
-              <div class="collected-product__actions">
```

Comment:
`.collected-product__actions` style can be removed.

Reply:
done